Firebase/Messaging/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2019-08-20 -- v4.1.3
+- [changed] Clean up the documents, unused macros, and folders. (#3490, #3537, #3556, #3498)
+- [changed] Update the header path to pod repo relative. (#3527)
```

Comment:
nit: changelogs should be past-tense. `Clean` -> `Cleaned`, `Update` -> `Updated`